app.js

```diff
@@ -0,0 +1,50 @@
+import createError from 'http-errors';
+import express, { json, urlencoded} from 'express';
+import cookieParser from 'cookie-parser';
+import logger from 'morgan';
+
+import YAML from 'yamljs';
+import swaggerUi from 'swagger-ui-express';
+
+import indexRouter from './routes/index.js';
+import ingredientsRouter from './routes/ingredients.js';
+import drinksRouter from './routes/drinks.js';
+import drinksingredientsRouter from './routes/drinksingredients.js';
+// import instructionsRouter from './routes/instructions.js';
+import reviewsRouter from './routes/reviews.js'
+
+const app = express();
+
+const swaggerDocument = YAML.load('./documentary/swagger-specs.yaml');
+app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
+
+
+app.use(logger('dev'));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+
+app.use('/', indexRouter);
+app.use('/api/drinks', drinksRouter);
+app.use('/api/ingredients', ingredientsRouter);
+app.use('/api/drinksingredients', drinksingredientsRouter);
+// app.use('/api/instructions', instructionsRouter);
```

Comment:
please when you want to remove something, just go ahead and remove it if not the code base is not going to look clean and hard to read.

Reply:
noted